url/interfaces.any.js

```diff
@@ -28,7 +21,7 @@
   USVString toJSON();
 };
 
-[Constructor(optional (USVString or URLSearchParams) init = ""),
+[Constructor(optional (sequence<sequence<USVString>> or USVString) init = ""), // or record<USVString, USVString>
```

Comment:
What's the error?  If it's a problem in parsing, a PR against w3c/webidl2.js seems to be in order (and it should be easy enough to fix, I'd think).  If it's an error from idlharness.js, that's what needs to get fixed.